### ISSUE TITLE
Some High Contrast Black issues

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#2619]
Pip search box still has different color for mouse over, and they are low contrast. Cookiecutter search box is ok.

![image](https://cloud.githubusercontent.com/assets/1696845/26653517/38b1224e-4608-11e7-8bbb-f18d5d0a3176.png)

Add virtual env combo box is white on blue instead of black on blue.
![image](https://cloud.githubusercontent.com/assets/1696845/26653563/4ce9face-4608-11e7-8d8e-fdda338a1dd0.png)

Environments window combo box is ok.
![image](https://cloud.githubusercontent.com/assets/1696845/26653590/63c2f5fc-4608-11e7-8625-5634bb1ac583.png)
